admin_manual/enterprise_clients/publishing_android_app.rst

```diff
@@ -0,0 +1,333 @@
+=======================================================
+Distributing Your Branded Android App (Enterprise Only)
+=======================================================
+
+Now that you have created your branded Android app with ownCloud's ownBuilder 
+service (:doc:`creating_branded_apps`) how do you distribute it to your users? 
+There are multiple ways: :ref:`email`, :ref:`publish_server`, or 
+:ref:`publish_google_play`.
+
+
+.. _email:
+
```

Comment:
I´d move the sign part to the top of the document. When we generate the branded clients, we provide to the customer 2 apks:
 - Debug one, this is for testing purporses, the device has to be set up for developing use, otherwise, the apk won´t be installed
- The unsigned one, this is the one that they have to sign.

No matter what distribution way the customer chooses, it has to be signed